.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Sunpy Community Chat Room
+    url: https://openastronomy.riot.im/#/room/#sunpy:openastronomy.org
```

Comment:
```suggestion
    url: https://openastronomy.element.io/#/room/#sunpy:openastronomy.org
```

Reply:
time to update the readme too

Reply:
yeah probably a lot of other places as well.

Reply:
Fixed.